csv_to_vanilla-i18n/csv_to_json.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const path= ""  // enter your data file path here
+const mycsv = fs.readFileSync(path,'utf-8');
+console.log(mycsv);
+
+const myjson = csvJSON(mycsv)
+console.log(myjson);
```

Comment:
Instead of logging, we are currently writing it in a directory named `vanilla-i18n`. Can we please follow the same pattern in this script as well?

Reply:
have resolved most of the comments , will do this later